src/client/telemetry/index.ts

```diff
@@ -330,6 +330,11 @@ export interface ISharedPropertyMapping {
      * For every DS telemetry we would like to know the type of Notebook Editor used when doing something.
      */
     ['ds_notebookeditor']: undefined | 'old' | 'custom' | 'native';
+    /**
+     * For every DS telemetry we would like to know whether the this is from AML compute or not.
+     * If not in AML compute, then do not send this telemetry.
+     */
+    ['isamlcompute']: 'yes';
```

Comment:
So we can identify this telemetry & exclude from others (as these forced optins into NativeNotebooks could mess up with experiments)

Reply:
So this part sounds like an unclean mess to me to have to deal with in telemetry. But as long as PMs have signed off on this I guess ok?

Reply:
Like, can't someone be an AML compute user and then also get put in the experiment? I guess we can code around in queries, but feels nasty.

Reply:
Agreed, and this is why they initially didn't want anyone to be opted into the experiment, but this is required & PMs are ok/happy with this.